packages/components/nodes/vectorstores/Postgres/Postgres.ts

```diff
@@ -58,28 +62,57 @@ class Postgres_VectorStores implements INode {
                 description: 'Keep track of the record to prevent duplication',
                 optional: true
             },
+            {
+                label: 'Driver',
+                name: 'driver',
+                type: 'options',
```

Comment:
`options`

Reply:
?

Reply:
the type should be `options`, i think previously it was `option`, but good now, fixed